pkg/cmd/root/extension_test.go

```diff
@@ -0,0 +1,146 @@
+package root_test
+
+import (
+	"io"
+	"testing"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/pkg/cmd/root"
+	"github.com/cli/cli/v2/pkg/extensions"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewCmdExtension_Updates(t *testing.T) {
+	tests := []struct {
+		name               string
+		extCurrentVersion  string
+		extIsPinned        bool
+		extLatestVersion   string
+		extName            string
+		extUpdateAvailable bool
+		extURL             string
```

Comment:
This doesn't seem to be used in any of the test cases. Should we remove it?

Reply:
Great catch, that's totally an oversight on my part and we should probably expand the `wantStderr` in the various test cases to check the url!

Reply:
Fixed!